src/NavigationDots/NavigationDots.tsx

```diff
@@ -50,14 +50,15 @@ export const NavigationDots = React.forwardRef<
     }))
   }, [activeDot, size])
 
-  const width = visibleDots.reduce(
-    (acc, el, index) =>
-      acc + (el.active ? 16 : el.small ? 6 : 8) + (index > 0 ? 4 : 0),
-    0
-  )
+  /*
```

Comment:
You can remove it altogether. The code is versioned anyway :smirk:

Reply:
that's an oversight ...